org.adempiere.base/src/org/compiere/model/MReference.java

```diff
@@ -108,7 +110,9 @@ public MReference(Properties ctx, MReference copy, String trxName) {
 	}
 	
 	/**	Reference Cache				*/
-	private static ImmutableIntPOCache<Integer,MReference>	s_cache = new ImmutableIntPOCache<Integer,MReference>(Table_Name, 20, 0, false, 0);
+	private static ImmutableIntPOCache<Integer,MReference>	s_cache = new ImmutableIntPOCache<Integer,MReference>(Table_Name, 20);

```

Comment:
Hi @nmicoud - why did you change the expire minutes from zero to default here?

Reply:
Hi @CarlosRuiz-globalqss 
Hum, I think there was some Chartreuse around :D 
Is probably a copy-paste error ; reverted